activation/activation.go

```diff
@@ -676,21 +617,27 @@ func (b *Builder) createAtx(ctx context.Context, challenge *types.NIPostChalleng
 	case <-b.syncer.RegisterForATXSynced():
 	}
 
+	client, err := b.postService.Client(b.nodeID)
+	if err != nil {
+		return nil, fmt.Errorf("get post client: %w", err)
+	}
+	info, err := client.Info(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("get post client info: %w", err)
+	}
```

Comment:
Or you could just save the metadata that comes in the response with the proof.

Reply:
I extended the `postClient.Proof` method to return a `types.PostInfo` object instead of a `types.PostMetadata` object. This removes the need to fetch info from the post service again after already having received the proof in (most) cases.

All cases will be covered in future PRs where I re-write ATX builder to work with multiple identities and not store data on disk any more.